debian/control

```diff
@@ -1,8 +1,8 @@
-Package: thelounge
+Package: lounge
 Version: 2.0.1-1
 Section: net
 Priority: optional
 Architecture: all
-Depends: nodejs (>= 4)
+Depends: nodejs (>= 4), bash
```

Comment:
No, I added it because something didn't work in the scripts. No idea why, and everyone has bash installed so meh.